src/stores/A_store.js

```diff
@@ -71,6 +71,53 @@ class A_store extends ReduceStore {
                 return {...state, stage, path, player: {x: nextLocation.x, y: nextLocation.y}};
             }
 
+            case ActionTypes.GENERATE_FOG: {
+                const {player, end} = state;
+                const {x, y} = player;
+                const revealedGrid = mapGenerator(player, end, new Grid(SIZE, SIZE));
+                const pathGrid = new Grid(SIZE, SIZE);
+
+                let visibleCells = revealedGrid.getVisible(x, y);
+                revealedGrid.obfuscateGrid(x, y, pathGrid, visibleCells);
+                let seenCells;
```

Comment:
why declare and then assign a couple lines later 🤔

Reply:
also doesn't look like it's reassigned

Reply:
🐇